templates/user/profile.tmpl

```diff
@@ -156,10 +161,14 @@
 					{{template "repo/user_cards" .}}
 				{{else if eq .TabName "followers"}}
 					{{template "repo/user_cards" .}}
-				{{else}}
+				{{else if or (eq .TabName "repositories") (not .ProfileReadme)}}
```

Comment:
Maybe `{{else if .ProfileReadme}}` here is better?

Reply:
See 8aede6e for fix if I understand your suggestion correctly.

Reply:
I mean could the last two conditions become?
```
{{else if .ProfileReadme}}
{{else}}
```